src/main/java/pl/debuguj/parkingspacessystem/controllers/ParkingSpaceController.java

```diff
@@ -0,0 +1,142 @@
+package pl.debuguj.parkingspacessystem.controllers;
```

Comment:
Podział pakietowy na "worki" z klasami a nie ze względu na tematykę - do tego wszystko jest publiczne

Reply:
Wydawało mi się, że podział jest w miarę dobry, bo wszystko rodzielone na usługi, wyjątki czy kontrolery. W sumie projekt dość mały więc podział wydawał się ok. Pakiet który mi się nie bardzo podobał to był "config". Co do "publiczności" klas zgadzam się, że nie zwróciłem na to uwagi.